web/src/lib/components/MetaTag.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
```

Comment:
The component should be called MetaTags (plural) since it inserts more than one tag!